actions/get-vault-secrets/action.yaml

```
@@ -0,0 +1,68 @@
+name: Get Vault Secrets
+description: Composite action (step) to get secrets from Grafana Labs' Vault instance.
+inputs:
+  repo_secrets:
+    description: |
+      Repository-specific secret mapping. Only Grafana Labs employees can list these secrets in Vault.
+      These secrets are stored in the `ci/data/repo/<repo owner>/<repo name>/<secret path>` path in Vault.
```

```suggestion
      These secrets are stored in the `ci/data/repo/grafana/<repo name>/<secret path>` path in Vault.
```

? maybe a simplification if it'll always be that